Library/Homebrew/formula.rb

```diff
@@ -923,9 +957,23 @@ def exec_cmd(cmd, args, out, logfn)
     exit! 1 # never gets here unless exec threw or failed
   end
 
+  def source_dir
+    if build.dsym?
+      prefix + "Source"
+    else
+      nil
+    end
+  end
```

Comment:
If I took this out then the debug info would all point to paths in the temporary directory, instead of where the files ultimately wind  up.   The user would have to explicitly tell the debugger where to find source.